app/models/chat_message.rb

```diff
@@ -68,44 +98,6 @@ def rebake!(invalidate_oneboxes: false, priority: nil)
 
     Jobs.enqueue(:process_chat_message, args)
   end
-
-  COOK_FEATURES = {
-    anchor: true,
-    "auto-link": true,
-    bbcode: true,
-    "bbcode-block": true,
-    "bbcode-inline": true,
-    "bold-italics": true,
-    "category-hashtag": true,
-    censored: true,
-    checklist: false,
-    code: true,
-    "custom-typographer-replacements": false,
-    "d-wrap": false,
-    details: false,
-    "discourse-local-dates": true,
-    emoji: true,
-    emojiShortcuts: true,
```

Comment:
No such markdown rule or custom markdown plugin. This is also controlled via a site setting so I don't know why this exists here.